<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowOptimization</class>
 <widget class="QWidget" name="WindowOptimization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Window Selection:</string>
     </property>
     <property name="buddy">
      <cstring>window_selection</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="window_selection">
     <item>
      <property name="text">
       <string>Shift Window Optimization</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Line Window Optimization</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="window_length_label">
     <property name="text">
      <string>Window Length:</string>
     </property>
     <property name="buddy">
      <cstring>window_length</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="window_length">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="offset_label">
     <property name="text">
      <string>Offset:</string>
     </property>
     <property name="buddy">
      <cstring>offset</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="offset">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Synthesis Method:</string>
     </property>
     <property name="buddy">
      <cstring>synthesis_method</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="synthesis_method">
     <item>
      <property name="text">
       <string>Transformation Based Synthesis</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Exact Synthesis</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Cost Function:</string>
     </property>
     <property name="buddy">
      <cstring>costs_function</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="costs_function">
     <item>
      <property name="text">
       <string>Gates</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Quantum Costs</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Transistor Costs</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Run-time:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="runtime">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
